Changing Prisma client output path causes import error

**Describe the bug**
When specifying a custom output path for Prisma client, the import statements in files generated by fabbrica cause errors.
These errors only occur when using **ES Modules** and setting 'module: node16' in tsconfig.json.
With these setttings, imports must include the full file path (not just directory), but fabbrica's generated imports only reference the directory.

**To Reproduce**
Steps to reproduce the behavior:

1. Create a project using **ES Modules** (`"type": "module"` in package.json)
2. Configure tsconfig.json with `"module": "node16"` 
3. In schema.prisma,
    - Specify some output path for Prisma client
    - Set `noTranspile = true` in the fabbrica generator config (see "Additional context" for behavior with `noTranspile = false`)
4. Run `npx prisma generate`
5. Try to compile with `npx tsc`

Error message:
```
src/generated/fabbrica/index.ts:1:27 - error TS2834: Relative import paths need explicit file extensions in ECMAScript imports when '--moduleResolution' is 'node16' or 'nodenext'. Consider adding an extension to the import path.

1 import type { User } from "../client";
                            ~~~~~~~~~~~
```

If I manually append `/index.js` or `/index.d.ts` to the end of the import path, the error will no longer occur.

**Expected behavior**
No error.

**Environment (please complete the following information):**

- Database: MySQL
- Node: 20.9.0
- @prisma/client: 6.5.0
- TypeScript: 5.8.1

**Your Schema**

```
generator client {
  provider = "prisma-client-js"
  output   = "../src/generated/client"
}

generator fabbrica {
  provider    = "prisma-fabbrica"
  output      = "../src/generated/fabbrica"
  noTranspile = true
}

datasource db {
  provider = "mysql"
  url      = env("DATABASE_URL")
}

model User {
  id Int @id @default(autoincrement())
}
```

**Your tsconfig.json**

```json
{
  "compilerOptions": {
    "target": "es2022",
    "module": "node16",
    "esModuleInterop": true,
    "strict": true,
    "skipLibCheck": true
  }
}
```

**Additional context**

Generated file:
https://gist.github.com/kyota-fujikura/e493fd6533353f62ffd29cdca2cd20ce

Note that by setting `noTranspile = false` the TypeScript compilation errors will disappear, but there's another problem:
the types of the objects created by the `build()` or `create()` method of fabbrica's factory class are not properly loaded (due to the same incorrect import path issue).
This results in no autocompletion or proper type checking when using these objects, just type names being displayed.

--- a/packages/prisma-fabbrica/src/getClientModuleSpecifier.test.ts
+++ b/packages/prisma-fabbrica/src/getClientModuleSpecifier.test.ts
@@ -5,11 +5,23 @@ describe(getClientModuleSpecifier, () => {
     { outputDirname: "dist", clientGeneratorOutputPath: undefined, expected: "@prisma/client" },
     { outputDirname: "dist", clientGeneratorOutputPath: "node_modules/@prisma/client", expected: "@prisma/client" },
     { outputDirname: "dist", clientGeneratorOutputPath: "node_modules\\@prisma\\client", expected: "@prisma/client" },
-    { outputDirname: "dist", clientGeneratorOutputPath: "prisma-client", expected: "../prisma-client" },
-    { outputDirname: "dist", clientGeneratorOutputPath: "../prisma-client", expected: "../../prisma-client" },
-    { outputDirname: "./dist", clientGeneratorOutputPath: "../prisma-client", expected: "../../prisma-client" },
-    { outputDirname: ".\\dist", clientGeneratorOutputPath: "..\\prisma-client", expected: "../../prisma-client" },
-    { outputDirname: "dist", clientGeneratorOutputPath: "..\\..\\prisma-client", expected: "../../../prisma-client" },
+    { outputDirname: "dist", clientGeneratorOutputPath: "prisma-client", expected: "../prisma-client/index.js" },
+    { outputDirname: "dist", clientGeneratorOutputPath: "../prisma-client", expected: "../../prisma-client/index.js" },
+    {
+      outputDirname: "./dist",
+      clientGeneratorOutputPath: "../prisma-client",
+      expected: "../../prisma-client/index.js",
+    },
+    {
+      outputDirname: ".\\dist",
+      clientGeneratorOutputPath: "..\\prisma-client",
+      expected: "../../prisma-client/index.js",
+    },
+    {
+      outputDirname: "dist",
+      clientGeneratorOutputPath: "..\\..\\prisma-client",
+      expected: "../../../prisma-client/index.js",
+    },
   ])(
     "outputDirname: $outputDirname, clientGeneratorOutputPath: $clientGeneratorOutputPath, expected: $expected",
     ({ outputDirname, clientGeneratorOutputPath, expected }) => {

--- a/packages/prisma-fabbrica/src/getClientModuleSpecifier.ts
+++ b/packages/prisma-fabbrica/src/getClientModuleSpecifier.ts
@@ -13,6 +13,6 @@ export function getClientModuleSpecifier(clientGeneratorOutputPath: string | und
     .normalize(clientGeneratorOutputPosixPath)
     .endsWith(["node_modules", "@prisma", "client"].join(path.sep))
     ? "@prisma/client"
-    : path.normalize("./" + path.relative(outputPosixDirname, clientGeneratorOutputPosixPath));
+    : path.normalize(path.join(".", path.relative(outputPosixDirname, clientGeneratorOutputPosixPath), "index.js"));
   return prismaClientModuleSpecifier;
 }

